ground/src/main/java/com/google/android/ground/ui/map/gms/FeatureClusterRenderer.kt

```diff
@@ -90,6 +124,11 @@ class FeatureClusterRenderer(
     cluster: Cluster<FeatureClusterItem>,
     markerOptions: MarkerOptions
   ) {
+    // Remove non-points before rendering cluster.
+    cluster.items
+      .map { it.feature }
+      .filter { it.geometry !is Point }
```

Comment:
Since we have geometry types other that Point, Polygon and MultiPolygon, would it be safer to check for equality rather than the other way around?

```
.filter { it.geometry is Polygon || it.geometry is Multipolygon }
```

Reply:
I think it's safer to only allow `Point` since that's the only type the clusters supports. If we do it the other way around the list would need to be exhaustive, and would need to be updated if we ever add a new geometry type. Wdyt?